docs/build/micro-rdk/movement-sensor/gyro-mpu6050.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Configure an MPU-6050 Gyroscope/Accelerometer"
+title: "Configure an MPU-6050 (Micro-RDK)"
```

Comment:
Was this a matter of exceeding the character limit?

Reply:
Yep